src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/PublicBookerService.kt

```diff
@@ -58,9 +65,21 @@ class PublicBookerService(
   fun validatePrisoner(bookerReference: String, prisonerNumber: String) {
     logger.trace("validate prisoner called for $prisonerNumber with booker reference $bookerReference")
     prisonVisitBookerRegistryClient.validatePrisoner(bookerReference, prisonerNumber)
+
+    // if no errors that means prisoner's prison and registered prison matches
+    // check if the prisoner's prison is supported on Visits
+    prisonerSearchClient.getPrisonerById(prisonerNumber).prisonId?.let { prisonId ->
+      if (!isPrisonSupportedOnVisits(prisonId)) {
```

Comment:
Does this belong in booker-registry as well? As it technically could be another returned 422 code validation error? 

Also is it possible for this to fail after validating the booker? how would their prison match but not be supported as we only store supported prison locations on the permitted prisoner table?

Reply:
the thinking is that the prison was enabled but for some reason later we decided to disable the prison for Public visits. A bit of an edge-case scenario.

Reply:
Would we still not want this validation in the same spot as all the other validation?

Reply:
as discussed, moved this check before the booker registry validation call